src/components/digitalMedia/DigitalMedia.tsx

```diff
@@ -1,182 +1,136 @@
-                    <Row className="digitalMedia_mediaBody">
+                <Container fluid className={`${styles.content} pt-5`}>
+                    <Row className="h-100">
                         <Col md={{ span: 10, offset: 1 }} className="h-100">
-                            <Row className="h-100">
-                                <Col md={{ span: 9 }} className="h-100">
-                                    <Row className="digitalMedia_titleBlock fw-bold pe-1">
-                                        <Col className="bg-primary-blue text-white">
-                                            {digitalMedia.id}
-                                        </Col>
-                                    </Row>
-                                    <Row className="digitalMedia_mediaBodyImages">
-                                        <Col md={{ span: 3 }}>
-                                            <DigitalMediaList />
-                                        </Col>
-
-                                        <Col md={{ span: 9 }} className="h-100">
-                                            <DigitalMediaFrame />
-                                        </Col>
-                                    </Row>
-                                    <Row>
-                                        <Col className="col-md-auto">
-                                            <button type="button" onClick={() => setObservationModalToggle(!observationModalToggle)}> 
-                                                Observation Annotation
-                                            </button>
-                                        </Col>
-                                    </Row>
-                                </Col>
-
-                                <Col md={{ span: 3 }} className="h-100 overflow-hidden">
-                                    <Tabs defaultActiveKey="properties">
-                                        <Tab eventKey="properties" title="Metadata">
-                                            <Row className="digitalMedia_mediaMetaContent overflow-scroll w-100 border border-dark">
-                                                <Col>
-                                                    <DigitalMediaProperties
-                                                        ToggleModal={(property: string) => ToggleModal(property)}
+                            <div className="h-100 d-flex flex-column">
+                                <Row className={styles.titleBar}>
+                                    <Col>
+                                        <TitleBar />
+                                    </Col>
+                                </Row>
+                                <Row className={`${styles.specimenContent} py-4 flex-grow-1 overflow-hidden`}>
+                                    <Col md={{ span: 3 }} className="h-100">
+                                        <IDCard ToggleModal={() => console.log('Toggle Modal')} />
```

Comment:
No, not really. It is more of a placeholder until I re-implented the annotations with the side bar, otherwise it will complain about it being an empty function. Will remove this in the annotation side bar PR.